pkgs/development/libraries/qt-5/5.12/default.nix

```diff
@@ -53,6 +53,7 @@ let
     qtbase = [
       ./qtbase.patch
       ./qtbase-fixguicmake.patch
+      ./qtbase-trayicons.patch
```

Comment:
Could you please add a comment here and below that these patches can be removed after Qt 5.12.4?

Reply:
Done :)